web-server/test/APIFunTests.scala

```diff
@@ -73,22 +73,10 @@ class APIFunTests extends PlaySpec with OneServerPerSuite {
   case class Uri(uri: String)
   case class Package(namespace: String, id: PackageId, uri: Uri, size: Long, checkSum: String, description: String,
                      vendor: String)
-  case class PackageResolver(id: PackageId, description: String, vendor: String)
   case class DeviceT(deviceName: String, deviceId: Option[String] = None, deviceType: String)
   case class FilterJson(namespace: String, name: String, expression: String)
   case class FilterPackageJson(filterName : String, packageName : String, packageVersion : String)
   case class ComponentJson(namespace: String, partNumber : String, description : String)
-  case class UpdateRequest(namespace: String, id: String, packageId: PackageId, creationTime: String,
```

Comment:
This isn't used anywhere, and the implicit decoders/encoders were generating one of those interpolation warnings.